Source/Charts/Data/Implementations/Standard/BarChartDataEntry.swift

```diff
@@ -102,17 +102,17 @@ open class BarChartDataEntry: ChartDataEntry
     
     @objc open func sumBelow(stackIndex :Int) -> Double
     {
-        if _yVals == nil
+        guard let yVals = _yVals else
```

Comment:
Throughout the project we're being inconsistent in how we write closures. Sometimes we open the closure on a new line, sometimes not, and sometimes we write one liners. Since you're going through a significant portion of the project already, can we consolidate on a format?

Reply:
I would love too, personally I think opening on the same line is fine